Source/ChilliSource/Rendering/Base/RenderSnapshot.h

```diff
@@ -191,6 +191,7 @@ namespace ChilliSource
         RenderCommandListUPtr m_preRenderCommandList;
```

Comment:
How are you handling the pre and post render commands? It is weird for RTT targets to contain pre or post commands, so it might be worth creating separate RTT snapshot objects? These could thing be included within the original snapshot when being passed down the render pipeline? Also ties in neatly with what I was saying about the RenderFrame; a single instance of each per frame.